Common/tests/TestBase/TaskTableTestBase.cs

```diff
@@ -554,20 +554,6 @@ public async Task CancelSessionAsyncShouldSucceed()
     }
   }
 
-  [Test]
-  public void CancelSessionAsyncShouldFail()
-  {
-    if (RunTests)
-    {
-      Assert.ThrowsAsync<SessionNotFoundException>(async () =>
-                                                   {
-                                                     await TaskTable!.CancelSessionAsync("NonExistingSessionId",
-                                                                                         CancellationToken.None)
-                                                                     .ConfigureAwait(false);
-                                                   });
-    }
-  }
```

Comment:
Why remove this test?

Reply:
This method does not throw `SessionNotFoundException` anymore.